dspace-server-webapp/src/main/java/org/dspace/app/rest/repository/ItemRestRepository.java

```diff
@@ -365,4 +375,53 @@ protected ItemRest createAndReturn(Context context, List<String> stringList)
         Item item = uriListHandlerService.handle(context, req, stringList, Item.class);
         return converter.toRest(item, utils.obtainProjection());
     }
+
+    /**
+     * Search method to find and return a paged list of duplicates, if the current user has authorization
+     *
+     * @param itemUuid the UUID of the item for which we are searching duplicates
+     * @param optionalPageable pagination options
+     * @return pagination potential duplicate results
+     */
+    @PreAuthorize("hasPermission(#itemUuid, 'ITEM', 'READ')")
+    @SearchRestMethod(name = "findDuplicates")
+    public Page<PotentialDuplicateRest> findDuplicates(@Parameter(value = "uuid", required = true) UUID itemUuid,
```

Comment:
this method is not appropriate for the ItemRestRepository. You will need eventually a dedicated RestRepository to expose the concept of "duplicates" where you can provide such search method

Reply:
I have moved this to a dedicated controller.

Reply:
yes, an oversight. removed.